README.md

```diff
@@ -1,2 +1,21 @@
 # celery-delayed-message
 Real celery delayed message 
+
+# Usage
+## install
+```shell
+pip install celery_delayed_message
+```
+## add schedule into your beat_schedule
+```
+{
+    'task': 'celery_delayed_message.redis_handlers.manager',
```

Comment:
task must support redis and RabbitMQ, so the name should not contains `redis`

Reply:
only redis broker need this